boms/pom.xml

```diff
@@ -33,4 +33,20 @@
     <module>upper-bounds-check</module>
   </modules>
 
+  <build>
```

Comment:
In previous commit of detaching parent pom from ones in `/boms`, configuration to skip maven-exec-plugin was missed.